### ISSUE TITLE
[Wayland][Nvidia] Crash on Wayland | Resolved in FF130

### ISSUE THREAD

**Issue by EndlessEden** [Alex313031/Mercury#230]
Mercury v129 is built against Firefox v129, which suffers from a segfaulting bug with wayland using nvidia drivers. This bug has been fixed in version 130+.

(bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1898476)

**Comment by Cybereu** [comment]
It's crashing so often that it's unusable.

**Comment by sheikh1233** [comment]
agreed, the next update should rebase it on at least 130 so that the segfaulting goes away
